### ISSUE TITLE
HTML report

### ISSUE THREAD

**Issue by MATBMS** [MATBMS/testimonials-slider#24]
### Inline style is not allowed

```html
<img class="avatar" id="t-avatar" src="./images/image-tanya.jpg" alt="Tanya Sinclair portrait" width ...
```

### "aria-label" cannot be used

"aria-label" cannot be used on this element.

```html
<div class="arrows" aria-label="Testimonials controls">
```

### Stop using inline styles

Using inline styles like `style="opacity: 1;"` which reduces maintainability and separates content from presentation. This causes harder updates and inconsistent styling across the site.

```html
<p class="quotes" style="opacity: 1;">“ I’ve been interested in coding for a while but never taken the jump, until now. I couldn’t recommend this course enough. I’m now in the job of my dreams and so excited about the future.”</p>
<div class="user-info">
  <p class="user-name" style="opacity: 1;">Tanya Sinclair</p>
  <p class="user-role" style="opacity: 1;">UX Engineer</p>
</div>
```

## To-dos

- [ ] inline style is not allowed
- [ ] "aria-label" cannot be used
- [ ] stop using inline styles
